packages/drivers/odsp-driver/src/index.ts

```diff
@@ -31,3 +31,8 @@ export * from "./odspDriverUrlResolver";
 
 // It's used by URL resolve code, but also has some public functions
 export * from "./odspFluidFileLink";
+
+// Wire format related
+export * from "./compactSnapshotParser";
+export * from "./odspUtils";
```

Comment:
The exported utils are in odspPublicUtils.ts. So, we can move the interface in this file.